src/main/java/sirius/biz/analytics/charts/explorer/UserAgentsTimeSeriesComputer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.analytics.charts.explorer;
+
+import sirius.biz.analytics.events.EventRecorder;
+import sirius.kernel.commons.Limit;
+import sirius.kernel.di.std.Part;
+
+import javax.annotation.Nullable;
+import java.time.LocalDate;
+import java.util.function.Function;
+
+/**
+ * A time series computer for the user agents data.
```

Comment:
```suggestion
 * Helps computing the user-agent distribution for a given Clickhouse event.
 * <p>
 * Note that the query has to be built manually, but helpful constants are provided below. Note that the query
 * has to yield the expected fields: <tt>year</tt>, <tt>month</tt>, <tt>day</tt>, and the values as created by
 * expressions as provided below.
 * <p>
 * Also note, that building SQL queries from multiple strings can be dangerous. Great care should be taken to
 * only add constant strings to the query. Everything else must be passed in as parameter via {@code  SQLQuery.set("key", value)}
 * 
 * @see DashboardUserAgentsChart
 */
```

Reply:
note, that the code should be edited in intellij so that all proper imports are available (also formatting might be boken)

Reply:
Done